### ISSUE TITLE
Clean up before release

### ISSUE THREAD

**Issue by simonthelwall** [publichealthengland/hcaidcs#1]
There are a bunch of warnings/errors in `devtools::check()`. These should all be cleaned up before release. 

 * data_compare `Error in sample.int(length(x), size, replace, prob) : 
  T used instead of TRUE`
 * `Data codoc mismatches from documentation object 'assignment_data'`
 * Undocumented arguments in documentation object 'ann_tab_pir_quarterly'
 * Undocumented arguments in documentation object 'fyq_to_fq'
 * Documented arguments not in \usage in documentation object 'fyq_to_fq'
 * There are a bunch of these: `aec_age_sex_plot: no visible binding for global variable
  'age_group_new'`

Consider moving sample data from apportion_prior_healthcare.R to data.R and document appropriately

**Comment by simonthelwall** [comment]
Also

```checking for code/documentation mismatches ... WARNING
Data codoc mismatches from documentation object 'assignment_data':
Variables in data frame 'assignment_data'
  Code: assignmentmethodcode finalpirassignedorganisation
        patient_category patientlocation pircasestatus
        provisionalorganisationname
  Docs: apportioned ccg_pir coca coha coia data_collection fmonth
        fyear6 hoha month_no month_string nhs_trust_pir org_code
        third_party_pir total_cases year_no

Data codoc mismatches from documentation object 'cdi_prior_hc_data':
Variables in data frame 'cdi_prior_hc_data'
  Code: apportioned ccg_pir coca coha coia data_collection fmonth
        fyear6 hoha month_no month_string nhs_trust_pir org_code
        phc_all_blank phc_unknown_3_mo third_party_pir total_cases
        year_no
  Docs: assignmentmethodcode finalpirassignedorganisation
        patient_category patientlocation pircasestatus
        provisionalorganisationname```

**Comment by simonthelwall** [comment]
Fixed (sufficiently) in a7e9d6f
